src/PostgREST/CLI.hs

```diff
@@ -96,16 +117,21 @@ readCLIShowHelp =
         <> O.short 'v'
         <> O.help "Show the version information"
 
-    exampleParser =
-      O.infoOption exampleConfigFile $
-        O.long "example"
-        <> O.short 'e'
-        <> O.help "Show an example configuration file"
+    exampleParser :: O.Parser CLI
+    exampleParser = 
+      CLIExample 
+        <$> showExample
+        <*> exampleConfig
+
+    exampleConfig =
+      O.strArgument $
+        O.metavar "ExampleType"
+        <> O.help "Type of config file/db/env"
```

Comment:
Doing this does not show `ExampleType` as a parameter of `--example` and it shows these options:

```
Usage: postgrest [-v|--version] 
                 [[-e|--example] ExampleType | 
                   [--dump-config | --dump-schema-cache] [FILENAME]]
```

When it should look something like:

```
Usage: postgrest-12.0.2 [-v|--version] [-e|--example EXAMPLETYPE]
                        [--dump-config | --dump-schema] [FILENAME]
```

You'll need to use directly `strOption` or `strArgument` (not sure which is better) and add `file` as a default value to avoid breaking changes. Here's an example on how to use this: https://www.fpcomplete.com/haskell/library/optparse-applicative/